05-condicionales/condicionales/condicionales.go

```diff
@@ -15,17 +15,19 @@ func Condicional(numero int) {
 	if numero%4 == 0 {
 		fmt.Printf("%v es divisible por 4\n", numero)
 	}
+
 	if numero < 0 {
 		fmt.Printf("%v es negativo\n", numero)
 	}
+
 	if (numero > 0 && numero < 10) || (numero < 0 && numero > -10) {
 		fmt.Printf("%v tiene un dígito\n", numero)
 	} else {
 		fmt.Printf("%v tiene varios dígitos\n", numero)
 	}
 }
 
-func Swtich_basico() {
+func SwtichBasico() {
```

Comment:
De acá [MixedCaps](https://go.dev/doc/effective_go#mixed-caps)

> Finally, the convention in Go is to use `MixedCaps` or `mixedCaps` rather than underscores to write multiword names.